packages/@dcl/inspector/src/components/Tree/Tree.tsx

```diff
@@ -152,7 +152,7 @@ export function Tree<T>() {
       const quitInsertMode = () => setInsertMode(false)
 
       const handleSelect = (event: React.MouseEvent) => {
-        if (event.type === ClickType.CONTEXT_MENU && event.ctrlKey) {
+        if (event.type === ClickType.CONTEXT_MENU && (event.ctrlKey || event.shiftKey)) {
```

Comment:
I tried using the `shift` key to multi select by clicking on the tree or elements on the renderer, and it doesn't seem to be working 🤔 `ctrl` is working fine.

Reply:
shift won't work since it's usual function is to select a range of items at once. Since we that's not required, I didn't want to have two different ways to do the same since it's confusing with the "normal" behaviour for selecting ranges